docs/reverse_proxy.md

```diff
@@ -46,7 +46,7 @@ when using a containerized Synapse, as that will prevent it from responding
 to proxied traffic.)
 
 Optionally, you can also set
-[`request_id_header`](../usage/configuration/config_documentation.md#listeners)
+[`request_id_header`](https://matrix-org.github.io/synapse/latest/usage/configuration/config_documentation.html?highlight=listeners#listeners)
```

Comment:
As a note, this would always link to the 'latest' version of the docs (even if you weren't browsing that one) and it would also not work when reading the documentation locally.
I think the style is fine but what was actually needed was fixing the path (`.` not `..`):
```suggestion
[`request_id_header`](./usage/configuration/config_documentation.md#listeners)
```